Framework/PythonInterface/mantid/api/src/Exports/IEventList.cpp

```diff
@@ -92,6 +93,15 @@ void export_IEventList() {
            "Get a vector of the weights of the events")
       .def("getPulseTimes", &IEventList::getPulseTimes, args("self"),
            "Get a vector of the pulse times of the events")
+      .def("getPulseTimesAsNumpy",
+           make_function(&IEventList::getPulseTimes,
+                         return_value_policy<Policies::VectorToNumpy>()),
+           "Get a vector of the pulse times of the events")
+      .def("getPulseTimesInNanosecondsFromUnixEpoch",
+           make_function(&IEventList::getPulseTimesInNanosecondsFromUnixEpoch,
+                         return_value_policy<Policies::VectorToNumpy>()),
+           "Get a vector of the pulse times in nanoseconds from the UNIX epoch "
+           "of the events")
```

Comment:
What is the difference here (apart from the epoch)? Is the former also a numpy array (of integers?)? If so, why is the latter needed?

Reply:
The first returns an array of `numpy.datetime64` the latter returns an array of `numpy.int64`.

What we want now will need the latter, what we may want later will probably need the former.

I already had the former implemented so just left it in, unless there is a strong reason to remove it I had planned to just leave it in.

Reply:
I am fine with leaving it. What I don't understand is how the former can be `numpy.datetime64` given that Mantid is using a different epoch?

I do wonder why we need the 2nd (new one) though. Can't numpy do the conversion to `int64`?

Reply:
> I am fine with leaving it. What I don't understand is how the former can be `numpy.datetime64` given that Mantid is using a different epoch?

The registered converter handles this.

> I do wonder why we need the 2nd (new one) though. Can't numpy do the conversion to `int64`?

Actually, yes. That would make more sense.